lib/bootsnap/cli.rb

```diff
@@ -131,6 +172,10 @@ def parser
           self.cache_dir = dir
         end
 
+        opts.on('--verbose', '-v', help.strip) do
+          self.verbose = true
```

Comment:
This is helpful to see what gets precompiled and adjust the `--exclude` patterns.

Reply:
🙏 definitely great feature. The help isn't properly set though it seems.